### PR TITLE
Webapp: Replace htop with pstree to show a process tree

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to replace htop with pstree in order to
dislay a tree of processes when a PR hang issue happens.

**Changes proposed in this PR:**
1. Added pstree.php
2. Removed htop.php

Signed-off-by: Geunsik Lim <leemgs@gmail.com>

---

### CODE REVIEW

File: ci/taos/webapp/pstree.php [hunk 1]
```diff
@@ -31,7 +30,8 @@
  */
 
 echo "========================================================";
-$output = shell_exec("echo q | htop | aha --black --line-fix > htop.html");
+$cmd = "pstree -p | aha --black --line-fix";
```
Comment (leemgs) [inline ci/taos/webapp/pstree.php]: Replace htop with pstree to show a tree of processes for debugging.

Comment (myungjoo) [reply ci/taos/webapp/pstree.php]: pstree does not show CPU / Memory usage, which is often useful. Do you intend to give up such information?

Comment (leemgs) [reply ci/taos/webapp/pstree.php]: The existing "PR: Resource Usage (top)" already shows  CPU and Memory usage. This PR is to show the running processes as a tree to monitor the relationship among the processes.

For example, "pstree -p'  is equal to 'htop + F5".

**Comment by leemgs** [comment]
* After this PR:

image

**Pull request merged**